addons/stock/product.py

```diff
@@ -555,23 +548,23 @@ def putaway_apply(self, cr, uid, putaway_strat, product, context=None):
                     categ = categ.parent_id
 
 
-class fixed_putaway_strat(osv.osv):
+class FixedPutawayStrat(models.Model):
     _name = 'stock.fixed.putaway.strat'
     _order = 'sequence'
-    _columns = {
-        'putaway_id': fields.many2one('product.putaway', 'Put Away Method', required=True),
-        'category_id': fields.many2one('product.category', 'Product Category', required=True),
-        'fixed_location_id': fields.many2one('stock.location', 'Location', required=True),
-        'sequence': fields.integer('Priority', help="Give to the more specialized category, a higher priority to have them in top of the list."),
-    }
 
+    putaway_id = fields.Many2one(comodel_name='product.putaway', string='Put Away Method', required=True)
+    category_id = fields.Many2one(comodel_name='product.category', string='Product Category', required=True)
+    fixed_location_id = fields.Many2one(comodel_name='stock.location', string='Location', required=True)
```

Comment:
If i am wrong then please correct me but ***comodel_name*** if not given then also its work fine :smiley: 
```
fixed_location_id = fields.Many2one('stock.location', string='Location', required=True)
```